modules/Bio/EnsEMBL/Analysis/Hive/Config/ShortncRNA.pm

```diff
@@ -0,0 +1,534 @@
+    dna_db_port               => '', # port for dna db host
+    repbase_logic_name        => '', # repbase logic name i.e. repeatmask_repbase_XXXX, ONLY FILL THE XXXX BIT HERE!!! e.g primates
+    release_number            => '' || $self->o('ensembl_release'),
+    species_name              => '', # e.g. mus_musculus
+    production_name           => '', # usually the same as species name but currently needs to be a unique entry for the production db, used in all core-like db names
+    taxon_id                  => '', # should be in the assembly report file
+    uniprot_set               => '', # e.g. mammals_basic, check UniProtCladeDownloadStatic.pm module in hive config dir for suitable set,
+    output_path               => '', # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+
+
+########################
+## Small ncRNAs params
+#########################
+    mirBase_fasta             => 'all_mirnas.fa', # What mirBase file to use. It is currently best to use on with the most appropriate set for your species
+    rfc_scaler                => 'filter_dafs_rfc_scaler_human.pkl',
+    rfc_model                 => 'filter_dafs_rfc_model_human.pkl',
+
+    # Clade-based filtering on rfam accessions
+    # Rfam db details should stay constant but check periodically
+    rfam_user => 'rfamro',
+    rfam_dbname => 'Rfam',
+    rfam_host => 'mysql-rfam-public.ebi.ac.uk',
+    rfam_port => 4497,
+
+    rfam_path => catfile($self->o('base_blast_db_path'), 'ncrna', 'Rfam_14.0'),
+    rfam_seeds => $self->o('rfam_path') . '/Rfam.seed',
+    rfam_cm => $self->o('rfam_path') . '/Rfam.cm',
+    filtered_rfam_cm => $self->o('ncrna_dir') .  '/Rfam.cm',
+    clade => $self->o('repbase_logic_name'),
+
+
+########################
+# Pipe and ref db info
+########################
+    pipe_db_name                  => $self->o('dbowner').'_'.$self->o('production_name').'_pipe_'.$self->o('release_number'),
+    dna_db_name                   => $self->o('dbowner').'_'.$self->o('production_name').'_core_'.$self->o('release_number'),
+
+    ncrna_db_server              => $self->o('databases_server'),
+    ncrna_db_port                => $self->o('databases_port'),
+    ncrna_db_name                  => $self->o('dbowner').'_'.$self->o('production_name').'_ncrna_'.$self->o('release_number'),
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    ensembl_release              => $ENV{ENSEMBL_RELEASE}, # this is the current release version on staging to be able to get the correct database
+
+
+    databases_to_delete => ['ncrna_db'],
+
+########################
+# BLAST db paths
+########################
+    base_blast_db_path        => $ENV{BLASTDB_DIR},
+    ncrna_blast_path          => catfile($self->o('base_blast_db_path'), 'ncrna', 'ncrna_2016_05'),
+    mirna_blast_path          => catfile($self->o('base_blast_db_path'), 'ncrna', 'mirbase_22'),
```

Comment:
```suggestion
    ncrna_blast_path          => catfile($self->o('base_blast_db_path'), 'ncrna', 'ncrna_2016_05'),
    mirna_blast_path          => catfile($self->o('base_blast_db_path'), 'ncrna', 'mirbase_22'),
```
These two should be catdir instead of catfile.